From f484a8a4cb4674e488a9ad422b6ebd08983213fe Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 21 Jun 2016 17:22:09 +0800
Subject: [PATCH] ARM64: dts: rk3399-evb2: remove 1.8G support for A72

Some RK3399 evb2 cannot support 1.8G for A72, maybe caused by
current limit, but remove it anyway for evb2.

Change-Id: Ibaa940696ccbdc59131c49e9a643a63863768ea2
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
index 35abddaf2195..57b16076efa7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
@@ -117,10 +117,10 @@
 		opp-hz = /bits/ 64 <1608000000>;
 		opp-microvolt = <1125000>;
 	};
-	opp07 {
-		opp-hz = /bits/ 64 <1800000000>;
-		opp-microvolt = <1200000>;
-	};
+//	opp07 {
+//		opp-hz = /bits/ 64 <1800000000>;
+//		opp-microvolt = <1200000>;
+//	};
 };
 
 &CPU_COST_A72 {
@@ -132,7 +132,7 @@
 		683   1850	/* 1200MHz */
 		805   2499	/* 1416MHz */
 		915   2922	/* 1608MHz */
-		1024  3416	/* 1800MHz */
+	//	1024  3416	/* 1800MHz */
 	>;
 	idle-cost-data = <
 	      15
@@ -165,7 +165,7 @@
 		683   1850	/* 1200MHz */
 		805   2499	/* 1416MHz */
 		915   2922	/* 1608MHz */
-		1024  3416	/* 1800MHz */
+	//	1024  3416	/* 1800MHz */
 	>;
 	idle-cost-data = <
 		 65
-- 
2.35.3

